.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: java
+sudo: required
 before_script:
 - unset _JAVA_OPTIONS
+- wget -O ~/gradle-5.1.1-bin.zip https://services.gradle.org/distributions/gradle-5.1.1-bin.zip
```

Comment:
Not fond of hardcoding the gradle version in the config. This means we'll need to keep updating it for the future. Looks like one other way to install latest version of gradle is using `sdk`. Can you investigate to see if that may be a good option for Travis to lay down the gradle version we need to run our test? Basically, we want to avoid hardcoding any version in the config.